### PR TITLE
#BAH-912 | Export multiple forms in form builder 

### PR DESCRIPTION
**Pull request by AlekhyaYalla**
Bahmni jira card link: https://bahmni.atlassian.net/browse/BAH-912

### CODE REVIEW

File: omod/src/main/java/org/bahmni/module/bahmni/ie/apps/controller/BahmniFormController.java [hunk 1]
```diff
@@ -19,70 +19,70 @@
...
+    }
 
-	@RequestMapping( value = baseUrl + "/export", method = RequestMethod.GET)
-	@ResponseBody
-	public ExportResponse export(@RequestParam List<String> uuids) {
-		return bahmniFormService.getFormsByListOfUuids(uuids);
-	}
+    @RequestMapping(value = baseUrl + "/export", method = RequestMethod.GET)
+    @ResponseBody
+    public ExportResponse export(@RequestParam ("uuid") List<String> uuids) {
```
Comment (bsneha90) [inline omod/src/main/java/org/bahmni/module/bahmni/ie/apps/controller/BahmniFormController.java]: Should the param be uuids?

File: api/src/test/java/org/bahmni/module/bahmni/ie/apps/service/impl/BahmniFormTranslationServiceImplTest.java [hunk 2]
```diff
@@ -542,7 +542,7 @@ public void shouldCopyTranslationsFromImportedForm() throws IllegalAccessExcepti
 		assertEquals(importedTranslationsJson, FileUtils.readFileToString(translationFile));
 	}
 
-	private static FormTranslation createFormTranslation(String locale, String version, String formName) {
+	public static FormTranslation createFormTranslation(String locale, String version, String formName) {
```
Comment (bsneha90) [inline api/src/test/java/org/bahmni/module/bahmni/ie/apps/service/impl/BahmniFormTranslationServiceImplTest.java:545]: Why has this been made public?

**Review by bsneha90 — CHANGES_REQUESTED** [review]

File: omod/src/main/java/org/bahmni/module/bahmni/ie/apps/controller/BahmniFormController.java (hunk 1 above)
Comment (AlekhyaYalla) [reply omod/src/main/java/org/bahmni/module/bahmni/ie/apps/controller/BahmniFormController.java]: yes changed

File: api/src/test/java/org/bahmni/module/bahmni/ie/apps/service/impl/BahmniFormTranslationServiceImplTest.java (hunk 2 above)
Comment (AlekhyaYalla) [reply api/src/test/java/org/bahmni/module/bahmni/ie/apps/service/impl/BahmniFormTranslationServiceImplTest.java:545]: We are using the same method in BahmniFormServiceImplTest.java, so for reusing it, we made it public.

Comment (bsneha90) [reply api/src/test/java/org/bahmni/module/bahmni/ie/apps/service/impl/BahmniFormTranslationServiceImplTest.java:545]: Can we move it to a util or helper class within test?

**Comment by bsneha90** [comment]
Can we resolve the conflicts on the PR?

**Pull request closed without merging**